app/views/documents/history.html.erb

```diff
@@ -33,7 +38,7 @@
             <% end %>
           </div>
 
-          <% if entry.internal_note? && entry.details %>
+          <% if entry.internal_note? && entry.details || entry.whitehall_migration? && entry.details.body %>
             <%= simple_format(escape_and_link(entry.details.body)) %>
           <% end %>
```

Comment:
Since this file is starting to get a bit hairy on logic I wonder if we want to have a partial that we pull in for rendering whitehall migration entries? - that could keep them somewhat distinct which will come with some pros of encapsulation and some cons for re-use (as we've done with internal note)

Reply:
I've moved the standard timeline entry and Whitehall timeline entries into separate partials.  The code looks much cleaner and has become easier to test.

Reply:
I've moved the standard timeline entry and Whitehall timeline entries into separate partials. The code looks much cleaner and has become easier to test.

(cc @alex-ju)

Reply:
Thanks for the heads up, Bruce. I'll probably wait with #1643 to get this one merged anyway – as I didn't manage to work on it at all today.